third_party/terraform/website/docs/r/google_project_service.html.markdown

```diff
@@ -14,6 +14,12 @@ Allows management of a single API service for an existing Google Cloud Platform
 For a list of services available, visit the
 [API library page](https://console.cloud.google.com/apis/library) or run `gcloud services list`.
 
+~> **Note:** `google_project_services` has been removed from the provider.
```

Comment:
The note can only be one paragraph, so this ends up looking a bit funny (I previewed this by  running `make website` in my tpg directory)
![Screen Shot 2020-05-21 at 12 05 06 PM](https://user-images.githubusercontent.com/1130694/82595997-6cecaa00-9b5b-11ea-8fa8-cc75787a95c8.png)

Reply:
I'd also recommend thinking about your audience for this block. Not everyone will realize that there had been a `google_project_services` resource in the past, so saying that it's been removed wouldn't mean anything to them.

Instead, I'd recommend phrasing like "Previous (pre-3.0.0) versions of the provider had a `google_project_services` resource, which was authoritative over the list [...]."

Reply:
done